README.md

```diff
@@ -19,7 +19,7 @@ This repository includes the CloudFormation template, Jupyter Notebook, and supp
 
 1. Choose **Launch Stack**:
 
-    [![Launch Stack](imgs/LaunchStack.jpg)](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?templateURL=https://aws-hcls-ml.s3.amazonaws.com/blog_post_support_materials/aws-alphafold/main/cfn.yaml)
+    [![Launch Stack](imgs/LaunchStack.jpg)](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?templateURL=https://cfn-without-vpc.s3.amazonaws.com/cfn-vpc.yaml)
```

Comment:
Where is this hosted?

Reply:
In a S3 bucket that was created, due to not use a public bucket :D

Reply:
how are you creating that bucket?

Reply:
manually for the moment, it can be built automatically either using cdk or cli, but as we are moving to cdk, I don't think that button/bucket is no longer needed. (Just modified in case of a client wants to have it)

Reply:
Please create a ticket to not lose tracking of this, besides of that LGTM

Reply:
Totally agree